### PR TITLE
fix: Customize benchmark db name via CLI arg

### LINKED ISSUE #20: Benchmark tool should not use `benchmark` as the db name
A user may have an existing `benchmark` db, the benchmark tool creates indexes and removes records inside of collections.

I suggest to change the `benchmark` db to `benchmark-${UNIXTIMESTAMP}`

Happy to create a PR for this

### PR DESCRIPTION
**Pull request by connorjburton**
Fixes #20 

Not sure why it's pulling commits through from my last branch when the they already merged. Let me know if that is an issue.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -69,7 +69,12 @@ papr.updateOne ~ 826.21 ops/sec
 mongoose.updateOne ~ 766.84 ops/sec
 ```
 
-This benchmark can be run locally with `yarn benchmark`, by default the benchmark tool will use `mongodb://localhost:27017`; however you can pass your own url by using the argument `--url` or `-u`.
+This benchmark can be run locally with `yarn benchmark`. See below for available arguments.
+
+| Parameter  | Accepted Type  | Default  | Description  |
+|---|---|---|---|
+| `--db` `-d`  | *string*  | `benchmark${UNIXTIME}`  | Name of the database  |
+| `--url` `-u`  | *string*  | `mongodb://localhost:27017`  | Mongo instance URL  |
```
Comment (avaly) [inline docs/README.md:77]: I'd rather remove these arguments from this main documentation page and instead only mention `yarn benchmark --help` (hoping it works).

File: benchmark/setup.js [hunk 2]
```diff
@@ -5,11 +5,13 @@ import mongoose from 'mongoose';
 import Papr from '../esm/index.js';
 
 const args = arg({
+  '--db': String,
+  '-d': '--db',
   '--url': String,
-  '-u': '--url'
+  '-u': '--url',
```
Comment (avaly) [inline benchmark/setup.js:11]: Please fix the eslint warning for these keys.

File: benchmark/setup.js [hunk 3]
```diff
@@ -5,11 +5,13 @@ import mongoose from 'mongoose';
...
 });
 
-const DATABASE = 'benchmark';
+const DATABASE = args['--db'] && args['--db'] ? args['--db'] : `benchmark${Date.now()}`;
```
Comment (avaly) [inline benchmark/setup.js:14]:
```suggestion
const DATABASE = args['--db'] && args['--db'] ? args['--db'] : `benchmark-${Date.now()}`;
```

**Review by avaly — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (connorjburton) [reply docs/README.md]: Now outputs as below. Can exit with a code other than `0` if you like.

Screenshot 2021-07-06 140908

**Review by avaly — APPROVED** [review]

**Comment by avaly** [comment]
@connorjburton Do you mind rebasing this branch on top of `main`?

**Comment by connorjburton** [comment]
Should be rebased now, thanks

**Pull request merged**